src/bin/tikv-server.rs

```diff
@@ -859,9 +860,11 @@ fn run_raft_server(pd_client: RpcClient,
     let pd_client = Arc::new(pd_client);
     let resolver = PdStoreAddrResolver::new(pd_client.clone())
         .unwrap_or_else(|err| exit_with_err(format!("{:?}", err)));
+    let snap_compression = get_fastest_supported_compression_type();
```

Comment:
Just call it at https://github.com/pingcap/tikv/pull/2001/files#diff-d73c1547780d463f5ca96c85a279da7cR1007 .